custom_components/wemportal/wemportalapi.py

```diff
@@ -705,6 +705,9 @@ def scrape_pages(self, response):
                         else:
                             unit = None
 
+                    if(name.endswith('leistungsanforderung')):
+                        unit = '%'
```

Comment:
there might be a better way to do this and if I recall correctly, if the lang is set to en the entity has even a different name (which should be added here as second clause then), but this code pins that value reliably to % for me.

Reply:
Looks good for now. If I find a better way I will incorporate it in the new minor release, which reworks some other things. 

Thank you for the contribution